.github/workflows/unittest.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Fix TFDS for older TF versions
         run: pip install tensorflow_datasets==3.2.*
         if: matrix.tf-version == '1.15.5' || matrix.tf-version == '2.0.4'
+      - name: Pin Keras for older TF versions
+        run: pip install keras==2.6.0
+        if: matrix.tf-version <= '2.6.0'
```

Comment:
Is this only a problem for 2.6 or others too?

Reply:
It is a problem with any TF version < 2.7 in combination with Keras 2.6. So by pinning to Keras 2.6 for older TF versions we should be fine.